pkgs/development/libraries/gtk/3.x.nix

```diff
@@ -94,6 +94,7 @@ stdenv.mkDerivation rec {
     "-Dgtk_doc=${boolToString withGtkDoc}"
     "-Dtests=false"
     "-Dtracker3=${boolToString trackerSupport}"
+    "-Dbroadway_backend=true"
```

Comment:
It would be nice if there was a conditional for this. In https://github.com/NixOS/nixpkgs/pull/106740 I noticed that for instance tracker support causes problems for an application packaged with autopatchelf and that uses gtk3, so I don't know if broadway_backend will cause issues in the future but you can never know.

Reply:
Done!